pkg/kopia/cli/internal/command/command.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2024 The Kanister Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package command
+
+import (
+	"github.com/kanisterio/safecli"
+
+	clierrors "github.com/kanisterio/kanister/pkg/kopia/cli"
+	"github.com/kanisterio/kanister/pkg/kopia/cli/internal/flag"
+)
+
+// Command is a CLI command/subcommand.
+type Command struct {
```

Comment:
Maybe we can call this `Subcommand` instead of `Command` for clarity?

Reply:
We have both commands and subcommands and Kopia also refers to them as commands AND subcommands.

```
$ kopia --help
usage: kopia [<flags>] <command> [<args> ...]

$ kopia repository --help
usage: kopia repository <command> [<args> ...]
Commands to manipulate repository.
```

but :)

```
$ kopia repository --help
usage: kopia repository <command> [<args> ...]
Commands to manipulate repository.

Flags:
  <...cut...>
Subcommands:  <--- subcommands here
  repository
    connect [<flags>]

```